hsctl

```diff
@@ -30,14 +30,15 @@ display_usage() {
 	echo "*** HydroShare Control script ***"
 	echo "usage: $0 loaddb          # loads database specified in hydroshare-config.yaml into running container"
 	echo "usage: $0 managepy [args] # use the arguments in a manage.py [args] call from the hydroshare container"
-	echo "usage: $0 update_index    # update solr index in a non-interactive way to be run as a cron job"
 	echo "usage: $0 rebuild         # deletes hydroshare container contents only and deploys using exsiting database"
 	echo "usage: $0 rebuild --db    # deletes all database and container contents and deploys from scratch"
 	echo "usage: $0 maint_off       # removes maintenance page (only if USE_NGINX = true)"
 	echo "usage: $0 maint_on        # displays maintenance page (only if USE_NGINX = true)"
+	echo "usage: $0 rebuild_index   # rebuild solr index in a non-interactive way to be run as a cron job"
```

Comment:
@mjstealey We don't need to rebuild_index to be run as a cron job, only need to run update_index at the moment, so the comment here for rebuild_index can be updated to remove "to be run as a cron job" Everything else looks fine to me. I will test it out in my local dev env now, and once everything works, I will merge it in.